src/features/VisualNovel/pages/ReleasesPage/ReleasesPage.tsx

```diff
@@ -1,24 +1,168 @@
-import React, { FC } from 'react';
+import React, { ReactElement, VFC } from 'react';
 import { useParams } from 'react-router';
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Link,
+  Text,
+  Tooltip,
+} from '@chakra-ui/react';
+import {
+  Language,
+  LanguageService,
+} from '../../../../api/services/languageService';
+import {
+  Platform,
+  PlatformService,
+} from '../../../../api/services/platformService';
+import { Icon } from '../../../../components/Icon/Icon';
+import { Release } from '../../../../models/release';
 import { useReleasesQuery } from '../../queries';
+import { ContentWrapper } from '../../../../components';
+import { ReleaseType } from '../../../../enums/releaseType';
+import { ReleaseService } from '../../../../api/services/releaseService';
+
+interface ReleaseGroups {
+  [language: string]: Release[];
+}
 
 /**
  * Releases page.
  */
-export const ReleasesPage: FC = () => {
+export const ReleasesPage: VFC = () => {
   const { id } = useParams();
 
   const {
     isLoading: isReleasesLoading,
+    data: releasesData,
+    error: releasesError,
   } = useReleasesQuery(id);
 
-  if (isReleasesLoading) {
-    return <div>Loading...</div>;
-  }
+  /**
+   * Groups releases by language.
+   * @param releases Releases.
+   */
+  const groupReleases = (releases: Release[]): ReleaseGroups =>
+    releases.reduce((accumulatedReleases, currentRelease) => {
+      const release = currentRelease.languages.reduce(
+        (accumulatedMultiLanguageRelease, currentLanguage) => {
+          const languageReleases = accumulatedReleases[currentLanguage] ?
+            [...accumulatedReleases[currentLanguage], currentRelease] :
+            [currentRelease];
+          return {
+            ...accumulatedMultiLanguageRelease,
+            [currentLanguage]: languageReleases,
+          };
+        },
+        {} as ReleaseGroups,
+      );
+      return { ...accumulatedReleases, ...release };
+    }, {} as ReleaseGroups);
```

Comment:
It's better to use generic instead of type assertion, like so `.reduce<ReleaseGroups>(`